_includes/overview/install.html

```diff
@@ -1,6 +1,30 @@
 <div class="c-section -intro">
   <div class="c-container">
     <h2 class="c-heading -h1">The Node.js API Framework</h2>
+    <div class="x-center">
+      <div class="v-block">
+        <a class="v-button">LoopBack 3 LTS</a>
```

Comment:
Since the `LoopBack 4` button goes to https://v4.loopback.io, I wasn't sure if this should be a link since it's already on the https://loopback.io page.